specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2019-02-02/table.json

```diff
@@ -68,6 +68,16 @@
           },
           {
             "$ref": "#/parameters/filterParameter"
+          },
+          {
+            "name": "NextTableName",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "format": "string",
```

Comment:
I don't think this is a valid format, if you want a simple string just use no format

Reply:
Fixed!